src/main/java/roomescape/theme/Theme.java

```diff
@@ -1,33 +1,50 @@
 package roomescape.theme;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Theme {
-    private Long id;
-    private String name;
-    private String description;
-
-    public Theme() {
-    }
-
-    public Theme(Long id, String name, String description) {
-        this.id = id;
-        this.name = name;
-        this.description = description;
-    }
-
-    public Theme(String name, String description) {
-        this.name = name;
-        this.description = description;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getDescription() {
-        return description;
-    }
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	@Column(name = "name", nullable = false)
+	private String name;
+	@Column(name = "description", nullable = false)
+	private String description;
+	@Column(name = "deleted", nullable = false, columnDefinition = "DEFAULT FALSE")
+	private boolean deleted;
+
+	protected Theme() {
+	}
+
+	private Theme(Long id, String name, String description, boolean deleted) {
+		this.id = id;
+		this.name = name;
+		this.description = description;
+		this.deleted = deleted;
+	}
+
+	public static Theme ofDeletedFalse(String name, String description) {
+		return new Theme(null, name, description, false);
+	}
```

Comment:
```suggestion
	public Theme(String name, String description) {
		this.name = name;
		this.description = description;
		this.deleted = false;
	}

	public static Theme ofDeletedFalse(String name, String description) {
		return new Theme(null, name, description, false);
	}
```

아래 메서드를 지우고 이렇게 하면 어떨까요?

Reply:
추가로 `ofDeletedFalse`라는 이름은 내부 구현이 잘 드러나는 네이밍인 것 같아요.
이 객체를 사용하는 입장에서는 논리 삭제로 구현되어 있는지 여부가 관심사가 아닐 수 있습니다!

Reply:
확실히 생성 시점에 내부 구현이 드러나는 네이밍인 것 같습니다. 
객체를 사용하는 입장에서 관심사.. 이 부분에 대해서 생각해보지 않았는데, 메소드 또한 같이 내부 구현을 드러내기 보다는 추상적인 네이밍으로 하는게 좋겠군요